ta4j-core/src/main/java/org/ta4j/core/num/DecimalNum.java

```diff
@@ -49,6 +49,8 @@
  */
 public final class DecimalNum implements Num {
 
+    private static final long serialVersionUID = 387498017007756808L;
```

Comment:
Any reason not to use '1L` for the initial UIDs?

Reply:
Great spot, updated to 1L.